firewall.conf

```diff
@@ -15,9 +15,9 @@ iptables -A INPUT -p tcp --dport ssh -j ACCEPT
 # Allow for all traffic from anywhere for the following services
 iptables -A INPUT -p tcp --dport 80 -j ACCEPT          # nginx
 iptables -A INPUT -p tcp --dport 443 -j ACCEPT         # nginx (SSL)
-iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
+#iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
```

Comment:
Perhaps we can add some notification around the button that tells that those features needs to be enabled by the cloud administrator?